decision-trees.Rmd

```diff
@@ -198,19 +198,17 @@ ggplot(mpg, aes(displ, cty, fill = drv)) +
 This looks considerably better, and although we still have some misclassification, that's ultimately probably a good thing because we don't want to start to overfit to our training data. Decision trees, like all models, should balance the bias-variance tradeoff. They are flexible and make few assumptions about the data, but that can also quickly lead to overfitting and poor generalizations to new data.
 
 ## Fitting a decision tree
-As an applied example, let's fit a decision tree model to data from the [2019 Data Science Bowl](https://www.kaggle.com/c/data-science-bowl-2019) from Kaggle. These data come from PBS KIDS on data from an educational gaming app called [Measure Up!](http://about.lunchbox.pbs.org/blogs/news/new-pbs-kids-measure-up-app-encourages-kids-and-parents-to-play-and-learn-together/). You can actually still submit to this competition (as of the time of this writing) if you'd like to see how performant your model is. The objective is to fit a model to predict kiddo's (ages 3-5) scores on an in-game assessment.
+As an applied example, let's fit a decision tree model to data from the [2019 Data Science Bowl](https://www.kaggle.com/c/data-science-bowl-2019) from Kaggle. These data come from PBS KIDS and are related to an educational gaming app called [Measure Up!](http://about.lunchbox.pbs.org/blogs/news/new-pbs-kids-measure-up-app-encourages-kids-and-parents-to-play-and-learn-together/). The objective is to fit a model to predict kids' (ages 3-5) scores on an in-game assessment.
 
 The outcome is `accuracy_group`, which is an ordered categorical variable ranging from 0-4. See the [Kaggle description](https://www.kaggle.com/c/data-science-bowl-2019/data) of the data for more information.
 
-### Load the data
-
-First, we need to read in the **train.csv** and **train_labels.csv** data. Note that I've removed the `event_data` column, which includes JSON data on `event_count`, `event_code`, and `game_time`, which are already represented as columns in **train.csv**. I've also sampled only read in the first 10,000 rows of the data so that things will run more quickly.
```

Comment:
@datalorax I removed this description since you don't show any of this data cleaning process now that we're using the `get_data()` approach (and because the reader probably won't need to know any of this, right?). I considered adding a footnote here to make it clear that the data has been slightly modified from its raw form on Kaggle and that we've selected only the first 10000 rows. But it looks like there are actually 86729 rows in this version of the data. Do you still want to sample only 10000 rows here?